### PR TITLE
[1.6.4] Use upstream loader

### PR DESCRIPTION
**Pull request by thecatcore**

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.2.1-bin.zip
```
Comment (BluCobalt) [inline gradle/wrapper/gradle-wrapper.properties:3]: Kinda weary about Gradle 7

Comment (thecatcore) [reply gradle/wrapper/gradle-wrapper.properties:3]: why?

Comment (BluCobalt) [reply gradle/wrapper/gradle-wrapper.properties:3]: as long as it works, I can't really complain

**Comment by BluCobalt** [comment]
Can you give a screenshot of it working in a development environment?

**Comment by thecatcore** [comment]
image

**Review by BluCobalt — APPROVED** [review]

**Review by hYdos — APPROVED** [review]

**Pull request merged**